KeychainExample/App.tsx

```diff
@@ -165,16 +169,16 @@ export default class KeychainExample extends Component {
         style={styles.container}
       >
         <View style={styles.content}>
-          <Text style={styles.title}>Keychain Example</Text>
+          <Text style={styles.title} onPress={() => Keyboard.dismiss()}>
```

Comment:
Seems odd, maybe makes more sense to wrap the whole thing in a scrollview to handle taps to dismiss?

Reply:
This is a quick hack to hide the keyboard for end-to-end tests. I wouldn't worry about it, but I can't find a better way to do it, if this is important

Reply:
Ah, makes sense. In a production app I would have cared, but not in a tester app 👍